.circleci/config.yml

```diff
@@ -47,3 +47,51 @@ jobs:
 
       - store_artifacts:
           path: doc/build/html
+
+      - persist_to_workspace:
+          root: doc/build
+          paths:
+            - html
+
+  deploy:
+    working_directory: ~/repo
+    docker:
+      - image: circleci/python:3.8.3-buster
+
+    steps:
+      - attach_workspace:
+          at: /tmp/build
+
+      - add_ssh_keys:
+          fingerprints:
+            - "e5:04:e6:c4:d6:c3:34:b9:02:e6:9f:25:1c:01:9e:e9"
+
+      - run:
+          name: install_dependencies
+          command: |
+            pip install --user ghp-import
+
+      - run:
+          name: upload_devdocs
+          command: |
+            set -e
+            git clone git@github.com:networkx/documentation.git
+            cd documentation/latest
+            cp -R /tmp/build/html/. .
+            git config --global user.email "networkx-circleci-bot@nomail"
+            git config --global user.name "networkx-circleci-bot"
+            git config --global push.default simple;
+            git commit -am "Docs built from $CIRCLE_SHA1"
+            ghp-import -n -f -p --remote=origin --branch=gh-pages .
```

Comment:
Does this do anything other than a git push?  I am not sure what using ``ghp-import`` buys us here.   And I am slightly worried it might do something other than a simple ``git push``.  I think we also need to delete the current content of ``documentation/latest``.  I would just do this:
```
git clone git@github.com:networkx/documentation.git
cd documentation
rm -rf latest
cp -R /tmp/build/html latest
git config --global user.email "networkx-circleci-bot@nomail"
git config --global user.name "networkx-circleci-bot"
git commit -am "Docs built from $CIRCLE_SHA1"
git push
```

Reply:
> Does ghp-import do anything other than a git push here? I am slightly worried it might do something other than a simple git push.

Good point, since this is all automated, there's no need for "convenience" tools - better to be explicit.

> I think we also need to delete the current content of documentation/latest. I would do this:

Another good point - I failed to consider *removing* documents.

Feedback incorporated in 3b27fe2.